src/main/scala/org/broadinstitute/dsde/firecloud/model/ElasticSearch.scala

```diff
@@ -9,7 +9,12 @@ import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
 
 case class AttributeDefinition(properties: Map[String, AttributeDetail])
 
-case class AttributeDetail(`type`: String, items: Option[AttributeDetail] = None, aggregate: Option[Boolean] = None)
+case class AttributeDetail(
+  `type`: String,
+  items: Option[AttributeDetail] = None,
+  aggregate: Option[Boolean] = None,
+  indexable: Option[Boolean] = Some(true)
```

Comment:
I'd prefer that `aggregate` and `indexable` be consistent: either both default to None, or both default to Some.